app/assets/javascripts/admin_layout.js

```diff
@@ -0,0 +1,10 @@
+//= require govuk_publishing_components/dependencies
+//= require govuk_publishing_components/all_components
+//= require jquery-ui.sortable.min
+//= require jquery_ujs
+//= require ./steps
+
+$(document).ready(function() {
+  GOVUKFrontend.initAll();
```

Comment:
This shouldn't be needed as the [JavaScript for component is being initialised in the gem](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/all_components.js#L6).

Reply:
In this app, it does not initialise the frontend. We have to trigger it within the `.ready()` function to initialise it. I believe it runs before it is rendered.